movies.js

```diff
@@ -53,16 +53,24 @@ exports.run = function (api, event) {
                                 error(api, event, err);
                                 return;
                             }
-                    var message = "Title: " + movie.title + "\n";
-                    message += "IMDb Rating: " + movie.imdb.rating + "\n";
-                    message += "RottenTomatoes: " + (movie.tomato == null ?  "N/A" : movie.tomato.meter + "%")+ "\n";
-                    message += "Metacritic: " + (movie.metacritic == null ? "N/A" : movie.metacritic) + "\n";
-                    api.sendMessage(message, event.thread_id);
 
+                            api.sendImage('url', movie.poster, '', event.thread_id);
+
+                            let imdbRating = movie.imdb ?  movie.imdb.rating : 'N/A',
```

Comment:
If you're going to start doing this then you need to update the entire file. Otherwise it's silly.